_docs/reference/api/enovy-config.md

```diff
@@ -0,0 +1,908 @@
+---
```

Comment:
This isn't really envoy config anyway, it's the istio proxy config (influenced by envoy).
So `proxy-config.md` would be a better name.